### DISCUSSION TITLE
Hide observation level attributes when JS is off

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1608] [Ideas]
**To discuss: should we hide observation-level attributes when JS is off so in line with when JS in on? or is there benefit in showing them?**

Table when JS is on:
<img src="https://user-images.githubusercontent.com/43404358/168612011-fc83e283-68db-458b-86cd-f5df243c70d6.png" width="50%" height="50%">

Table when JS if off:
<img src="https://user-images.githubusercontent.com/43404358/168612687-1f3dd2e2-8e36-4a11-b882-504ebf08650c.png" width="75%" height="75%">

**Comment by LucyGwilliamAdmin** [comment]
@phillipgardiner @brockfanning @otis-bath thoughts please!

Think useful to show Units and Series in this scenario as otherwise wouldn't be able to differentiate:
![image](https://user-images.githubusercontent.com/43404358/168618028-2581255b-1d3a-45f0-bd0f-29b51fa0be69.png)

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin @phillipgardiner @otis-bath To play devil's advocate - could it be considered a "progressive enhancement" that the obs-level attributes are hidden? I ask because I think it would be technically difficult to hide those columns. (Probably not impossible, just not immediately obvious how it would happen.)

**Comment by otis-bath** [comment]
@LucyGwilliamAdmin @phillipgardiner @brockfanning completely agree we need **Units** and **Series** showing, I don't hate seeing the observation level attributes, know its not perfect but does provide more information from the data that doesn't look really out of place, if this is going to be a labour intensive fix not sure having these columns is that bad?

**Comment by phillipgardiner** [comment]
@otis-bath @phillipgardiner @LucyGwilliamAdmin I tend to agree with all the above comments. I am no expert on this but my basic understanding is that at its simplest we can display all the data available and then use 'progressive enhance'  in order to only show those columns that are ultimately required.

**Comment by LucyGwilliamAdmin** [comment]
@otis-bath @phillipgardiner @brockfanning I'm also in agreement - we will leave it as it is then
